html/index.html

```diff
@@ -1,26 +1,27 @@
 <!DOCTYPE html>
 <head>
     <title>My Personal Website</title>
-    <style></style>
 </head>
 <body>
     <header>
-        <img src="/media/cc0-images/grapefruit-slice-332-332.jpg">
+        <img alt="Grapefruit slice atop a pile of other slices" src="/media/cc0-images/grapefruit-slice-332-332.jpg">
         <nav class="quicknav">
             <a href="#News">News</a>
             <a href="#contact">Contact</a>
             <a href="#about">About</a>
        </nav>
-        <div class = "searchbar">
-          <input type="text" placeholder="Search..">
-        </div>
+        <form>
+            <li class="button">
```

Comment:
You won't use a `li` element without a `ul` or `ol` element. If you want to style the button you can add the class selector to the `button` element directly.